### DISCUSSION TITLE
Concerns regarding case #163: "court of law is in location X"

### DISCUSSION THREAD

**Discussion by madoleary** [tosdr/edit.tosdr.org#1032] [Ideas]
This case is attached to 16 contributions but adds no value, as is. 

I presume the purpose of the case was to distinguish between jurisdictions that are legally "unfriendly" to user data protection and those that are more "friendly." In its current form, it does not reflect this separation, the points regarding unfriendly and friendly jurisdictions appear to be mixed, and the rating is set at a constant "neutral." 

I propose we create two cases: 1) The jurisdiction(s) governing the terms of service is more favorable to user data protection, 2) The jurisdiction(s) governing the terms of service is less favorable to user data protection.

**Comment by Vinnl** [comment]
My interpretation is that there was no value judgement intended with this case, but merely to inform. In other words: points that are linked to this case indicate where the actual court of law for that service is, without providing a statement on whether that's a good or a bad place to be in.

That's not to say that your proposed cases are not a good idea additionally, though, although I am a bit wary of potential problems arising from more nuanced distinctions. For example, in the UK you can currently benefit from GDPR, but they're also part of five eyes - so would being in the UK be a good thing or a bad thing?

**Comment by madoleary** [comment]
I get what you're saying, but there are existing points on the website
where I think an additional value judgment needs to be made:
https://edit.tosdr.org/points/1095

It seems to me that that deserves a higher rating?

A compromise would be to keep the existing case, and add the two others.
Then just let the users decide from there.

**Comment by Vinnl** [comment]
Yes, I think so too, you have my blessing :)

**Comment by michielbdejong** [comment]
sure, you can make a case for "jurisdiction is within the EU for EU users", and we could rate that as a  'Good'.
